06.bowling_object/lib/game.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  attr_reader :frames
+
+  def initialize(shot_pins)
+    @frames = build_frames(shot_pins)
```

Comment:
build_framesの中でインスタンス変数を宣言、代入しちゃってるので、今の実装だとこう書いても同じですね。

```suggestion
    build_frames(shot_pins)
```

build_framesの中ではインスタンス変数を触らないようにしましょう

Reply:
インスタンス変数を触らず、引数を渡すように修正しました